### PR TITLE
Streamline Repo Onboarding and Management

### PR DESCRIPTION
**Pull request by nirvanatikku**
* Added a Makefile for operations
* Refactored app into `app` alongside `api` and `resources` (was `samples`)
* Updates to Data Access Handler bootstrap script
* DLGaaS: Added support for editing the CLIENT_DATA
* DLGaaS: Fix the unnecessary user_text at start
* Updated README and Quick Start

### CODE REVIEW

**Review by sunil-thorat — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,298 +44,250 @@ For more information on how to leverage the client across various environments,
...
-⚠️ Ensure you are in the virtual environment when running. (ref. `source env/bin/activate`)
+### Option 2: Run Natively
 
-##### Securely (Recommended)
+Run the two processes.
 
+App:
 ```bash
-func start --useHttps --cert ../certificate.pfx --password "<REPLACE_ME>"
+make native-app-run-secure
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply README.md:236]: Fixed

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+	rm -rf app/public
+
+	@echo 2/2: Cleaning APIs
+	rm -rf api/.venv
+	rm -rf api/__pycache__
+
+# Utiltiies
+launch: certs-setup containers-run
+	@echo Launch https://localhost:8000
+	$(XOPEN) https://localhost:8000
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply Makefile:139]: Fixed

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+	cp resources/local.settings.json api/local.settings.json
+	cd api/; \
+	python3 -m venv .venv; \
+	source .venv/bin/activate; \
+	pip install -r requirements.txt
+
+native-run-app-secure: native-build-app
+	cd app/; \
+	npm run develop -- --https --cert-file ../resources/localhost+2.pem --key-file ../resources/localhost+2-key.pem
+	open https://localhost:8000
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply Makefile:109]: Fixed

File: README.md [hunk 4]
```diff
@@ -44,298 +44,250 @@ For more information on how to leverage the client across various environments,
...
 ```
 
-By default, the Functions should be served on `https://localhost:7071`.
-
-##### Insecurely
-
+API:
 ```bash
-func start
+make native-api-run-secure
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply README.md:241]: Fixed

File: Makefile [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+
+# Native 
+native-build-app:
+	cd app/; npm install
+
+native-build-api:
+	cp resources/local.settings.json api/local.settings.json
+	cd api/; \
+	python3 -m venv .venv; \
+	source .venv/bin/activate; \
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply Makefile:103]: Fixed

File: Makefile [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+	@echo Preparing requirements for certificates.
+	$(XPACKAGEMANAGER) install openssl mkcert
+
+certs-setup:
+	@echo Creating and Installing Certs for SSL
+	cd resources; \
+		$(XMKCERT) localhost 127.0.0.1 ::1; \
+		$(XMKCERT) -install
+
+	@read -s -p "Password: " password; \
```
[the remark this answers is not in the document]
Comment (nirvanatikku) [reply Makefile:62]: Added a comment for now, can look to streamline in future update

**Pull request merged**